src/Encamina.Enmarcha.AspNet.Mvc/Bindings/CustomDateTimeModelBinderProvider.cs

```diff
@@ -13,7 +13,7 @@ public class CustomDateTimeModelBinderProvider : IModelBinderProvider
     /// <summary>
     /// Initializes a new instance of the <see cref="CustomDateTimeModelBinderProvider"/> class.
     /// </summary>
-    public CustomDateTimeModelBinderProvider() : this(null)
+    public CustomDateTimeModelBinderProvider() : this(null!)
```

Comment:
In this case, I would make `customDateTimeFormat` nullable (along with all necessary changes) and remove this `!`